### ISSUE TITLE
Training NaNs immediately, if BatchNormalization is used

### ISSUE THREAD

**Issue by Xonxt** [ildoonet/tf-pose-estimation#509]
Have you (or anyone else) stumbled into the problem, that if you're including the BatchNormalization in your Depthwise Convolutions (or "Separable Convolutions" in your case, but that's pretty much the same thing), the training immediately goes to NaN after the first several steps of the first epoch? No matter, which learning rate I choose or which optimizer I'm using, it always happens.

Now, if I just remove Batch Normalizations, then the training proceeds fine and I'm getting acceptable inference results, but it's weird.

Two things to note: I'm using Keras instead of pure Tensorflow, and I'm also training and inferring on depth images instead of RGB.
